Core.js

```diff
@@ -603,7 +603,9 @@ Core = {
         return stack;
     }
     , processGlobal: function() {
-        CatchEvent(DOM_Init);
+        if (typeof window != 'undefined') {
+            CatchEvent(DOM_Init);
+        }
```

Comment:
This change does not make sense, because "if" will not affect event subscription